sequencer/NetworkPlan.py

```diff
@@ -255,11 +280,15 @@ def network(self):
         return self._network
     
     @property
+    def original_metrics(self):
+        """returns the original (unprocessed) metrics data_frame"""
+        return self._original_metrics
```

Comment:
to keep consistent with _network and _metrics.  Better ideas welcome.  
I don't like original_metrics being there at all, but the dependency in sequencer was hard to get rid of quickly.  Will address that with large refactoring or rewrite.